pallets/runtime/tests/src/asset_test.rs

```diff
@@ -1938,198 +1909,6 @@ fn can_set_primary_issuance_agent_we() {
     assert_eq!(Asset::token_details(ticker), token);
 }
 
-#[test]
```

Comment:
Is it worth keeping this, but commenting it out?

Reply:
In general, I am not a fan of commented code. It can always be fetched from git history if needed :).